.github/workflows/unit-tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
```

Comment:
Since node 16 was released a few weeks ago, node 10 is EOL so I've updated our actions to include v16 and remove v10

Reply:
I'm not so sure about removing Node v10 just yet, because Frontity still supports it.